ReactNative/ReactTwitter/core/views/splash-screen.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import {
+  StyleSheet,
+  View,
+  Text,
+  Navigator
+} from 'react-native'
+var AuthenticationService = require('../service/authentication-service.js')
+
+var SplashScreenView = React.createClass({
+  propTypes: {
+    navigator: React.PropTypes.instanceOf(Navigator).isRequired
+  },
+
+  getInitialState () {
+    return {
+      authenticationService: new AuthenticationService()
+    }
+  },
+
+  componentDidMount () {
+    let authService = this.state.authenticationService
+
+    authService.loadDataFromDisk()
+      .then(() => {
+        return new Promise((resolve, reject) => {
+          setTimeout(resolve, 1100)
```

Comment:
it was carefully chosen for a good UX impact 👅 
I will extract it to a constant